pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -294,6 +294,7 @@ public ServerType getServerType() {
   }
 
   public static class Controller {
+    public static final String PREFIX_OF_CONFIG_OF_SEGMENT_FETCHER_FACTORY = "pinot.controller.segment.fetcher";
```

Comment:
Can we move it to `ControllerConf.java`?

Reply:
Talked to Jack offline, I agree that we should either put all controller-specific configs in controllerconf, server specific in server conf, etc, or put them all in common constants. Will create a separate ticket for that. I want that refactor to be separate from fixing incorrect configurations such as this one.